protocol/query_and_mutate_response.go

```diff
@@ -0,0 +1,109 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at
+ *          http//license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package protocol
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/oceanbase/obkv-table-client-go/util"
+)
+
+type ObTableQueryAndMutateResponse struct {
+	ObUniVersionHeader
+	ObPayloadBase
+	affectedRows   int64
+	affectedEntity *ObTableQueryResponse
```

Comment:
I guess ``queryResult`` or ``entity`` is a better name
```suggestion
	queryResult *ObTableQueryResponse
```
```suggestion
	queryResult *ObTableQueryResponse
```

Reply:
agree